client/src/components/AddItemFormSteps/StepOne.jsx

```diff
@@ -67,14 +68,24 @@ const StepOne = ({
   const handlePhotoChange = (photos) => {
     if (photos.length >= 3 && photos.length <= 5) {
       setImages(photos);
+
+      const selectedFiles = [];
+      const targetFiles = photos;
+      const targetFileObject = [...targetFiles];
```

Comment:
You can omit targetFiles, as it is used only here.
const targetFileObject = [...photos];